arduino-ide-extension/src/browser/widgets/component-list/component-list-item.tsx

```diff
@@ -14,30 +14,14 @@ export class ComponentListItem<
       )[0];
       this.state = {
         selectedVersion: version,
-        focus: false,
-        versionUpdate: false,
       };
     }
   }
 
-  override componentDidUpdate(
-    prevProps: ComponentListItem.Props<T>,
-    prevState: ComponentListItem.State
-  ): void {
-    if (this.state.focus !== prevState.focus) {
-      this.props.onFocusDidChange();
-    }
-  }
-
   override render(): React.ReactNode {
     const { item, itemRenderer } = this.props;
     return (
-      <div
-        onMouseEnter={() => this.setState({ focus: true })}
-        onMouseLeave={() => {
-          if (!this.state.versionUpdate) this.setState({ focus: false });
-        }}
-      >
+      <div>
```

Comment:
Having a `div` here is superfluous at this point. Please replace it with a react fragment (`<>...</>`).

Reply:
I replaced it [here](https://github.com/arduino/arduino-ide/pull/1568/commits/496f9be9fb78077d7422ec32050f92d574a75dd7). Thanks 👍

Reply:
even the fragment is superfluous we can just return the node no?
```
return itemRenderer.renderItem(
    Object.assign(this.state, { item }),
    this.install.bind(this),
    this.uninstall.bind(this),
    this.onVersionChange.bind(this)
);

```